releasenotes/notes/remove_aer_circuit_from_metadata-e4fe09029c1a3a3c.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Results of `backend.run()` were not serializable because they include `AerCircuit`s.
+    This commit makes the results serializable by removing `AerCircuit`s from metadata.
```

Comment:
```suggestion
    Results of ``backend.run()`` were not serializable because they include :class:`.AerCircuit`\ s.
    This commit makes the results serializable by removing :class:`.AerCircuit`\ s from metadata.
```